src/waveform/renderers/glwaveformrenderbackground.cpp

```diff
@@ -0,0 +1,25 @@
+#include "waveform/renderers/glwaveformrenderbackground.h"
+
+GLWaveformRenderBackground::GLWaveformRenderBackground(
+        WaveformWidgetRenderer* waveformWidgetRenderer)
+        : WaveformRenderBackground(waveformWidgetRenderer) {
+}
+
+GLWaveformRenderBackground::~GLWaveformRenderBackground() {
+}
```

Comment:
empty virtual destructor. Just out of curiosity, is there ever a need for one? If not, why is it here?

Reply:
because I copied waveformrenderbackground.cpp and it's there because I couldn't be bothered cleaning up code that isn't mine ;-) These empty destructors are all over the place. But I'll remove this one.

Reply:
done